src/components/presale/presale.tsx

```diff
@@ -286,127 +329,181 @@ class Presale extends Component<PRESALEPROPS, PRESALESTATE> {
     const disabled =
       !(this.state.isOpen && this.state.connected) ||
       this.state.waiting ||
-      !this.state.inputValid;
+      !this.state.inputValid ||
+      !this.state.termsChecked;
 
     const claimDisabled = !this.state.connected || this.state.tokenLocked <= 0;
-
-    const investLimit = Math.min(
-      this.state.ethUser,
-      3.0 - this.state.ethInvested
-    );
+    const failureClass = this.state.inputValid ? '' : ' pcr-input-failure';
 
     const { t } = this.props;
 
     return (
-      <div className="tk-aktiv-grotesk-condensed presale-main presale-column">
+      <div className="tk-grotesk-lightbold presale-main presale-column">
```

Comment:
classname `presale-column` is unused in the css

Reply:
thx, fixed